website/docs/docs/core/connect-data-platform/risingwave-setup.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "RisingWave setup"
+id: "risingwave-setup"
+description: "Read this guide to learn about how to set up RisingWave in dbt."
+meta:
+  maintained_by: RisingWave
+  pypi_package: 'dbt-risingwave'
+  authors: 'Dylan Chen'
+  github_repo: 'risingwavelabs/dbt-risingwave'
+  min_core_version: 'v1.6.1'
+  min_supported_version: ''
+  cloud_support: Not Supported
+  slack_channel_name: '#all-things-risingwave'
+  slack_channel_link: 'https://www.risingwave.com/slack'
+  platform_name: 'RisingWave'
+  config_page: '/reference/resource-configs/risingwave-configs'
```

Comment:
Let's see if this is causing the build to fail because it does not exist yet
```suggestion
 <!--  config_page: '/reference/resource-configs/risingwave-configs' -->
```